package.json

```diff
@@ -30,12 +30,12 @@
     "Other"
   ],
   "keywords": [
-    "dependencies",
+    "security",
     "vulnerability",
-    "license",
+    "dependency scan",
     "node",
     "java",
-    "multi-root ready"
```

Comment:
keep "multi-root ready", IIRC this would enable keywords to be pickedup from `contributes.languages[*].aliases`

Reply:
Okay :+1

Reply:
Okay :+1:

Reply:
@arajkumar this is something different right?
https://code.visualstudio.com/docs/editor/multi-root-workspaces